x-pack/plugins/observability/public/pages/slos/components/badges/slo_badges.tsx

```diff
@@ -6,67 +6,27 @@
  */
 
 import React from 'react';
-import { EuiBadge, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { EuiFlexGroup } from '@elastic/eui';
 import { SLOWithSummaryResponse } from '@kbn/slo-schema';
-import { i18n } from '@kbn/i18n';
 
-import { useKibana } from '../../../../utils/kibana_react';
 import { SloIndicatorTypeBadge } from './slo_indicator_type_badge';
 import { SloStatusBadge } from '../../../../components/slo/slo_status_badge';
 import { SloTimeWindowBadge } from './slo_time_window_badge';
-import { toAlertsPageQueryFilter } from '../../helpers/alerts_page_query_filter';
-import { paths } from '../../../../config/paths';
 import type { ActiveAlerts } from '../../../../hooks/slo/use_fetch_active_alerts';
+import { SloActiveAlertsBadge } from './slo_active_alerts_badge';
 
 export interface Props {
   slo: SLOWithSummaryResponse;
   activeAlerts?: ActiveAlerts;
 }
 
 export function SloBadges({ slo, activeAlerts }: Props) {
-  const {
-    application: { navigateToUrl },
-    http: { basePath },
-  } = useKibana().services;
-
-  const handleClick = () => {
-    if (activeAlerts) {
-      navigateToUrl(
-        `${basePath.prepend(paths.observability.alerts)}?_a=${toAlertsPageQueryFilter(
-          activeAlerts
-        )}`
-      );
-    }
-  };
-
   return (
     <EuiFlexGroup direction="row" responsive={false} gutterSize="s" alignItems="center">
       <SloStatusBadge slo={slo} />
-      <EuiFlexItem grow={false}>
-        <SloIndicatorTypeBadge slo={slo} />
-      </EuiFlexItem>
-      <EuiFlexItem grow={false}>
-        <SloTimeWindowBadge slo={slo} />
-      </EuiFlexItem>
-      {!!activeAlerts && (
-        <EuiFlexItem grow={false}>
-          <EuiBadge
-            iconType="warning"
-            color="danger"
-            onClick={handleClick}
-            onClickAriaLabel={i18n.translate(
-              'xpack.observability.slo.slo.activeAlertsBadge.ariaLabel',
-              { defaultMessage: 'active alerts badge' }
-            )}
-            data-test-subj="o11ySlosPageSloActiveAlertsBadge"
-          >
-            {i18n.translate('xpack.observability.slo.slo.activeAlertsBadge.label', {
-              defaultMessage: '{count, plural, one {# alert} other {# alerts}}',
-              values: { count: activeAlerts.count },
-            })}
-          </EuiBadge>
-        </EuiFlexItem>
-      )}
+      <SloIndicatorTypeBadge slo={slo} />
+      <SloTimeWindowBadge slo={slo} />
+      <SloActiveAlertsBadge activeAlerts={activeAlerts} />
```

Comment:
📣 I've moved the FlexItem into the component itself, and added a new component for the active alerts badge 🧹